src/www/ui/template/user_edit.html.twig

```diff
@@ -41,6 +41,10 @@
         <th width="25%">{{ "Select the user's access level."|trans }}</th>
         <td>{{ macro.select('user_perm', allAccessLevels, 'user_perm', accessLevel) }}</td>
       </tr>
+      <tr class="classic">
+        <th width="25%">{{ "Select the user's clearing role."|trans }}</th>
+        <td>{{ macro.select('is_clearing_admin', allClearingPerms, 'is_clearing_admin', clearingPerm) }}</td>
+      </tr>
```

Comment:
Do we need to assign this roles from edit user now?

Reply:
Ah. this needs to be removed. will do it.